stm32-modules/heater-shaker/include/heater-shaker/messages.hpp

```diff
@@ -87,8 +87,8 @@ struct ErrorMessage {
 */
 struct GetTemperatureResponse {
     uint32_t responding_to_id;
-    int16_t current_temperature;
-    uint32_t setpoint_temperature;
+    double current_temperature;
```

Comment:
same here on the double, is it really necessary? I know this type is generally more expensive than others.

Reply:
Doubles aren't really truly necessary, but it's nice to just not have to worry about it. And the timescales for the temperature stuff is so much slower than anything else we care about that it'll be fine.